### PR TITLE
Documentation for /counters endpoint. (AB-7771)

### PR DESCRIPTION
**Pull request by yuriykurylenko**
Relates to https://autobutler.atlassian.net/browse/AB-7771 and https://github.com/autobutler/autobutler/pull/6301.

### CODE REVIEW

File: source/includes/_counters.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+> To get the numbers of direct incoming jobs and jobs that should be completed, use this code:
+
+```shell
+curl -X "GET" "https://www.autobutler.dk/api/v2/mechanics/counters" \
+     -H "Authorization: token"
+```
+
+```javascript
+jQuery.ajax({
+  url: "http://dev.autobutler.dk:3000/api/v2/mechanics/counters",
```
Comment (dipth) [inline source/includes/_counters.md]: Wrong url

File: source/includes/_counters.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+> The above command returns JSON structured like this:
+
+```json
+{
+  "directIncomingJobsCount": 5,
+  "jobsShouldBeCompletedCount": 3
+}
+```
+
+This endpoint provides the numbers of direct incoming jobs and jobs that should be completed by current mechanic:
```
Comment (dipth) [inline source/includes/_counters.md]: We use the term "the workshop" instead of "current mechanic".

Comment (dipth) [inline source/includes/_counters.md]: Same below...

File: source/includes/_counters.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+### HTTP Request
+
+`GET https://www.autobutler.dk/api/v2/mechanics/counters`
+
+### Response JSON
+
+Attribute                        | Type          | Can be blank? | Description
+-------------------------------- | ------------- | ------------- | -----------------------------------------------------------------------
+directIncomingJobsCount          | integer       | no            | Number of direct incoming jobs for current mechanic
+jobsShouldBeCompletedCount       | integer       | no            | Number of jobs that are most likely completed (based on dates)
```
Comment (dipth) [inline source/includes/_counters.md]: This description seems a bit misleading... Shouldn't it read something like this instead?
"Number of jobs that the workshop needs to mark as complete"

**Comment by yuriykurylenko** [comment]
@dipth as agreed with @andreaseriksson, counters have been moved to jobs_controller, documentation updated correspondingly.

File: source/includes/_jobs.md [hunk 4]
```diff
@@ -114,3 +114,63 @@ offer      | object  | yes           | A <a href="#offerdetails">offerDetails</a
...
+> To get the numbers of direct incoming jobs and jobs that should be completed, use this code:
+
+```shell
+curl -X "GET" "https://www.autobutler.dk/api/v2/mechanics/jobs/counters" \
+     -H "Authorization: token"
+```
+
+```javascript
+jQuery.ajax({
+  url: "http://dev.autobutler.dk:3000/api/v2/mechanics/jobs/counters",
```
Comment (dipth) [inline source/includes/_jobs.md]: Please fix the url to point to the production server via https

**Pull request closed without merging**